tests/validation/tests/v3_api/test_pipeline.py

```diff
@@ -0,0 +1,55 @@
+from .common import *  # NOQA
+from .test_app import cluster_and_client as target_from_id_and_client
+import pytest
+
+#RANCHER_CLUSTER NAME, CATTLE_API_URL, ADMIN_TOKEN
+namespace = {"p_client": None, "ns": None, "cluster": None, "project": None}
+
+@pytest.fixture(scope='module', autouse="True")
+def create_project_client(request):
+    client, cluster = get_global_admin_client_and_cluster()
+    create_kubeconfig(cluster)
+    p, ns = create_project_and_ns(
+        ADMIN_TOKEN, cluster, random_test_name("testworkload"))
+    p_client = get_project_client_for_token(p, ADMIN_TOKEN)
+    namespace["p_client"] = p_client
+    namespace["ns"] = ns
+    namespace["cluster"] = cluster
+    namespace["project"] = p
+
+    def fin():
+        client = get_admin_client()
+        for ns_x in get_cluster_client_for_token(namespace["cluster"], ADMIN_TOKEN). \
+            list_namespace(projectId=namespace["project"].id): 
+            client.delete(ns_x)
+        client.delete(namespace["project"])
+        
+    request.addfinalizer(fin)
+
+def check_condition(status):
+    def _find_condition(resource):
+        if not hasattr(resource, "lastRunState"):
+            return False
+
+        if resource.lastRunState is None:
+            return False
+
+        if resource.lastRunState == status:
+            return True
+        return False
+    return _find_condition
+
+def create_example_pipeline():
+    return namespace["p_client"].create_pipeline(
+        name="test-" + random_str(),
+        repositoryUrl="https://github.com/rancher/pipeline-example-go.git",
+        triggerWebhookPr=False,
+        triggerWebhookPush=False,
+        triggerWebhookTag=False)
+
+def test_run_example_pipeline():
```

Comment:
Let's add some validations around the artifacts that get deployed from this example pipeline and viewing the logs from a stage.